git_aggregator/main.py

```diff
@@ -89,6 +89,16 @@ def get_parser():
         type=_log_level_string_to_int,
         nargs='?',
         help='Set the logging output level. {0}'.format(_LOG_LEVEL_STRINGS))
+
+    main_parser.add_argument(
+        '-e', '--expand-env',
```

Comment:
Per @pedrobaeza in https://github.com/Tecnativa/git-aggregator/commit/50467a388c419a65e0c5e8eeb5dca01b76742f8a#commitcomment-20682952:
> this is an incorrect use of parse arguments.